python/ray/tests/test_cancel.py

```diff
@@ -587,6 +587,27 @@ def verify():
     wait_for_condition(verify)
 
 
+@pytest.mark.parametrize("use_force", [True, False])
+def test_cancel_with_dependency(shutdown_only, use_force):
+    ray.init(num_cpus=4)
+
+    @ray.remote(num_cpus=1)
+    def inner():
+        while True:
```

Comment:
[nit] can we change this to more descriptive way:

1. instead of busy sleeping every 0.1s we can sleep longer like 1000s
2. don't return 1, since it won't return anyway
3. make a more descriptive func name than `inner`

Reply:
makes sense, updated